sympy/combinatorics/homomorphisms.py

```diff
@@ -421,27 +421,31 @@ def block_homomorphism(group, blocks):
     H = GroupHomomorphism(group, codomain, images)
     return H
 
-def group_isomorphism(G, H, isomorphism=True, epimorphism=False, all=False):
+def find_homomorphism(G, H, injective=True, surjective=True, compute=True, all=False):
```

Comment:
I think `injective` and `surjective` should be `False` by default because it would be more natural to say `find_homomorphism(..., injective=True)` instead of `find_homomorpism(..., surjective=False)` is you were looking for an injection. You could define `group_isomorphism` to use as a shortcut though